Clients/GUI/Scripts/SignUper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using GUI.GUIModels;
+using System.Net;
+using DTO;
+using System.IO;
+using System.Text.Json;
+
+namespace GUI.Scripts
+{
+    public class SignUper
+    {
+        public static HttpStatusCode SignUp(SignUpViewModel insert)
+        {
+            var output = new UserEmailPassword(insert.Email, insert.Password);
```

Comment:
If you have output , why don't you named parameters of method "input"? 
Also, it isn't obvious. What concrete insert do you gave to this method. Do I have always to see which type of this variable it is ? Words "insert" and "output " not understandable at all

Reply:
i corrected that

Reply:
Hm. You corrected just first line. Input and output also not so understandable. 

And still: why do you continue to return HttpStatusCode? Maybe it would be beter if we will make it also void and to throw exceptions here? Why do you have to check it in another method